spec/services/user_registration_spec.rb

```diff
@@ -0,0 +1,108 @@
+require 'spec_helper'
+
+describe UserRegistration do
+
+  describe "#register" do
+    context "with valid credit card" do
+      let(:stripeToken) { 'abc123' }
+      let(:amount) { 999 } 
+      let(:charge) { double(:charge, successful?: true) }
+      before do
+        StripeWrapper::Charge.should_receive(:create).and_return(charge)
```

Comment:
`.should_receive` is actually the old syntax. New would be `expect(StripeWrapper::Charge).to receive(:create).and_return(charge)`